YYMM-office-hours-recap-template.md

```diff
@@ -0,0 +1,19 @@
+Title: "<Title Here>" - DEI WG Community Office Hour -
+
+Body:
+<Heading>
```

Comment:
We can probably drop this section since I think Summary, Topics Discussed and Learnings Observed are enough.